### PR TITLE
Implement SRC-AP-VB3 in the differ template

### PR DESCRIPTION
**Pull request by cosletmihai**
- updated current sections
- added a bunch of new sections
- update and add test data

### CODE REVIEW

File: resources/eds_templates/diff_report/templates/main.html [hunk 1]
```diff
@@ -16,75 +16,84 @@
...
-    {% include "sections/added_property_concept_scope_note.html" with context%}
-    {% include "sections/deleted_property_concept_scope_note.html" with context%}
+    {% include "sections/notes/added_property_concept_skos_definition.html" with context %}
+    {% include "sections/notes/deleted_property_concept_skos_definition.html" with context %}
+    {% include "sections/notes/added_property_concept_scope_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_scope_note.html" with context %}
+    {% include "sections/notes/added_property_concept_example.html" with context %}
+    {% include "sections/notes/deleted_property_concept_example.html" with context %}
 
     <h1>Other notes</h1>
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/main.html:60]: otehr notes shall then include the subsections available under "Notes" headding

File: resources/eds_templates/diff_report/templates/main.html [hunk 2]
```diff
@@ -16,75 +16,84 @@
...
-    {% include "sections/added_property_concept_history_note.html" with context%}
-    {% include "sections/deleted_property_concept_history_note.html" with context%}
-
-    {% include "sections/added_property_concept_skos_note.html" with context%}
-    {% include "sections/deleted_property_concept_skos_note.html" with context%}
-
+    {% include "sections/notes/added_property_concept_skos_change_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_skos_change_note.html" with context %}
+
+    <h1>Semantic Properties</h1>
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/main.html:64]: we shall have here *skos:narrower* as well

File: resources/eds_templates/diff_report/templates/main.html [hunk 3]
```diff
@@ -16,75 +16,84 @@
...
+    {% include "sections/concept_evolution/added_property_concept_owl_deprecated.html" with context %}
+    {% include "sections/concept_evolution/deleted_property_concept_owl_deprecated.html" with context %}
+    {% include "sections/concept_evolution/added_property_dct_is_replaced_by.html" with context %}
+    {% include "sections/concept_evolution/deleted_property_dct_is_replaced_by.html" with context %}
+    {% include "sections/concept_evolution/added_property_euvoc_start_date.html" with context %}
+    {% include "sections/concept_evolution/deleted_property_euvoc_start_date.html" with context %}
+    {% include "sections/concept_evolution/added_property_euvoc_end_date.html" with context %}
+    {% include "sections/concept_evolution/deleted_property_euvoc_end_date.html" with context %}
+
+    <h1>Notes</h1>
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/main.html:88]: merge with otehr notes section above

File: resources/eds_templates/diff_report/templates/main.html [hunk 4]
```diff
@@ -16,75 +16,84 @@
...
+    {% include "sections/notes/added_property_concept_editorial_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_editorial_note.html" with context %}
+    {% include "sections/notes/added_property_concept_history_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_history_note.html" with context %}
+    {% include "sections/notes/added_property_concept_skos_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_skos_note.html" with context %}
+
+    <h1>Miscellaneous Properties</h1>
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/main.html:96]: merge with version info section above

File: resources/eds_templates/diff_report/templates/sections/added_reified_concept_notation_value.html [hunk 5]
```diff
@@ -29,19 +29,12 @@ <h2 class="ui header">Added concept reified notations</h2>
 PREFIX sd: <http://www.w3.org/ns/sparql-service-description#>
 PREFIX sh: <http://purl.org/skos-history/>
 PREFIX xhv: <http://www.w3.org/1999/xhtml/vocab#>
-SELECT DISTINCT ?instance (str(?prefLabel) as ?instanceLabel) (?value AS ?addedValue)
+SELECT DISTINCT ?instance (str(?prefLabel) as ?instanceLabel) (str(?value) AS ?addedValue) ?languageTag
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/sections/added_reified_concept_notation_value.html:32]: no language tag in notations pls. Only in labels and notes (both reified and simple)

File: resources/eds_templates/diff_report/templates/sections/deleted_reified_concept_notation_value.html [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 <h2 class="ui header">Deleted concept reified notations</h2>
 <section class="ui basic segment">
-    <p>The table below lists the deleted concept <strong>skos:notation</strong> values
+    <p>The table below lists the deleted concept <strong>skos:notation/rdf:value</strong> values
```
Comment (costezki) [inline resources/eds_templates/diff_report/templates/sections/deleted_reified_concept_notation_value.html:5]: nice indication

**Review by costezki — APPROVED** [review]
well done

File: resources/eds_templates/diff_report/templates/main.html (hunk 2 above)
Comment (cosletmihai) [reply resources/eds_templates/diff_report/templates/main.html:64]: `skos:narrower` is not defined in the `src-ap` document

**Pull request merged**